### ISSUE TITLE
method addRow and addRows is updating

### ISSUE THREAD

**Issue by crisdanilo** [theoephraim/node-google-spreadsheet#302]
The methods `addRow` and `addRows` is updating a row. That methods should add a row.

The solution is to change the value in  `insertDataOption` to `INSERT_ROWS` in  `addRows`. However, you can use `axios.request`of the **GoogleSpreadsheetWorksheet**  API, setting the value and adding the values. Ex:

```javascript
const response = await doc.axios.request({
      method: 'post',
      url: `/values/${encodeURIComponent(this.a1SheetName)}:append`,
      params: {
        valueInputOption: 'USER_ENTERED', // RAW
        insertDataOption: 'INSERT_ROWS', // INSERT_ROWS
        includeValuesInResponse: true,
      },
      data: {
        values: [data, data, data, data, data],
      },
    });
```

I send PR to fix de bug.

Thanks!!

**Comment by crisdanilo** [comment]
How can I contribute to the project?. I would like to make a PR but I have permission denied @theoephraim

**Comment by theoephraim** [comment]
Well you can't push to this repo directly, but you should be able to open a PR - ie fork the repo, create a branch with your changes, then open a PR to merge it back into this repo. If you're getting permission denied when trying to push to your own fork of this repo, it likely has to do with general permission issues and not being logged in fully on your machine. Sometimes this can be due to github 2FA.

**Comment by sapics** [comment]
@crisdanilo I made a PR in #305 which you proposed here, I guess. Could you check the fix?

**Comment by theoephraim** [comment]
now available as options - see the comments in #305
